.travis.yml

```diff
@@ -1,42 +1,131 @@
-# Mac OS X (https://github.com/travis-ci-tester/toolchain-table)
+# OSX/Linux (https://github.com/travis-ci-tester/toolchain-table)
 
 language:
-    - objective-c
+  - cpp
+
+os:
+  - linux
+  - osx
+
+# Container-based infrastructure (Linux)
+# * https://docs.travis-ci.com/user/migrating-from-legacy/#How-can-I-use-container-based-infrastructure%3F
+sudo:
+  - false
+
+# Install packages differs for container-based infrastructure
+# * https://docs.travis-ci.com/user/migrating-from-legacy/#How-do-I-install-APT-sources-and-packages%3F
+# * http://stackoverflow.com/a/30925448/2288008
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - python3
+
+      # python3-pip package is not available, use 'easy_install3':
+      # * https://github.com/travis-ci/apt-package-whitelist/issues/768
+      - python3-setuptools # easy_install3
+
+      # https://github.com/travis-ci-tester/travis-test-clang-cxx-11
+      - libstdc++-4.8-dev
+
+      # https://github.com/travis-ci-tester/travis-test-gcc-cxx-11
+      - g++-4.8
+
+      # For Qt
+      - libegl1-mesa-dev
+
+      # Packages for Android development: http://superuser.com/a/360398/252568
+      - libncurses5:i386
+      - libstdc++6:i386
+      - zlib1g:i386
 
 env:
-    - TOOLCHAIN=libcxx CONFIG=Release
-    - TOOLCHAIN=libcxx CONFIG=Debug
-    - TOOLCHAIN=xcode CONFIG=Release
-    - TOOLCHAIN=xcode CONFIG=Debug
-    - TOOLCHAIN=ios-nocodesign-arm64 CONFIG=Release
-    - TOOLCHAIN=ios-nocodesign-arm64 CONFIG=Debug
-    - TOOLCHAIN=ios-nocodesign-armv7 CONFIG=Release
-    - TOOLCHAIN=ios-nocodesign-armv7 CONFIG=Debug
-    # - TOOLCHAIN=analyze CONFIG=Release (error in OpenCV)
-    # - TOOLCHAIN=analyze CONFIG=Debug
+  - TOOLCHAIN=libcxx CONFIG=Release
+  - TOOLCHAIN=libcxx CONFIG=Debug
+  # FIXME (no OpenCV) {
+  # - TOOLCHAIN=clang-libstdcxx CONFIG=Release
+  # - TOOLCHAIN=clang-libstdcxx CONFIG=Debug
+  # }
+  - TOOLCHAIN=osx-10-9 CONFIG=Release
+  - TOOLCHAIN=osx-10-9 CONFIG=Debug
+  # FIXME (glew, glfw3) {
+  # * https://travis-ci.org/ruslo/gatherer/jobs/133200753
+  # - TOOLCHAIN=gcc-4-8 CONFIG=Release
+  # - TOOLCHAIN=gcc-4-8 CONFIG=Debug
+  # }
+  # FIXME (no OpenCV) {
+  # - TOOLCHAIN=ios-nocodesign-wo-armv7s CONFIG=Release
+  # - TOOLCHAIN=ios-nocodesign-wo-armv7s CONFIG=Debug
+  # }
```

Comment:
Not sure I understand the question. I need to upload binary cache of OpenCV from local macine because of Travis timeouts. This already done for `ios-nocodesign`, need to do the same for `ios-nocodesign-wo-armv7s`. Will do when have access to OSX machine back.